pegasus/sites.v3/code.org/views/top_hoc_tutorials.haml

```diff
@@ -1,7 +1,7 @@
 = view :top_hoc_tutorial_applab, image_match_youtube_height: false
 
 - outbreak_image_url = CDO.code_org_url("/images/outbreak/outbreak.png")
-- minecraft_img_url = CDO.code_org_url("/shared/images/fill-485x260/courses/logo_tall_aquatic.jpg")
+- minecraft_img_url = CDO.code_org_url("/shared/images/fill-485x260/courses/logo_tall_mc_escape.png")
```

Comment:
This is used on https://code.org/hourofcode/overview (see screenshot in PR description)